packages/nextjs/src/utils/userIntegrations.ts

```diff
@@ -1,33 +1,31 @@
 import { Integration } from '@sentry/types';
 
-export type UserFunctionIntegrations = (integrations: Integration[]) => Integration[];
-export type UserIntegrations = Integration[] | UserFunctionIntegrations;
+export type IntegrationsFunction = (integrations: Integration[]) => Integration[];
+export type UserIntegrations = Integration[] | IntegrationsFunction;
 
-type Options = {
-  [integrationName: string]:
-    | {
-        keyPath: string;
-        value: unknown;
-      }
-    | undefined;
-};
+/** Pairs of a property's path within an integration object (ex: `options.xyz`) and the desired value for that property */
+type IntegrationProperties = Array<[string, unknown]>;
 
 /**
  * Recursively traverses an object to update an existing nested key.
  * Note: The provided key path must include existing properties,
- * the function will not create objects while traversing.
+ * as the function will not create objects while traversing.
```

Comment:
I wonder if it's worth it to wrap in `try/catch` then, as it may blow up.

Reply:
Yeah it will in the current implementation. That’s why I specified BrowserTracing before.

Reply:
Done.